test/unit/spread.spec.ts

```diff
@@ -4,6 +4,7 @@ import { formatCode } from "../util";
 
 // TODO: Make some utils for testing other targets
 const expectUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toMatch(/[^.]unpack\(/);
+const expectArg: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toMatch(/ arg/);
```

Comment:
what's with this space before arg in the pattern? That doesn't seem right, also this pattern seems way too easy to accidentally pass, you should probably use more specific patterns, like 'unpack(arg)' for example

Reply:
It looks like I used this pattern for testing (before we had bindings to execute 5.0) and forgot to remove it. It was supposed to check for translation to `= arg`, which is correct for 5.0, rather than `= {...}`, as is used for the other versions. It was also extremely crude and, clearly, should never have been committed...

I've reinstated the unpack test because verifying the translation is redundant; if it was wrong, it wouldn't execute or match the JS result. See https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/4ab5891c38f84773ef3e70c7b35985581267c7c1.